app/helpers/documents_v2_helper.rb

```diff
@@ -1,2 +1,15 @@
 module DocumentsV2Helper
+  def codap_v2_link(codap_server, runAsGuest=false, fromLaunchPage=false)
+    data = {
+      "documentServer" => root_url(protocol: 'https')
+    }
+    data["runAsGuest"] = 'true' if runAsGuest
+    data["launchFromLara"] = 'true' if fromLaunchPage
```

Comment:
I thought the client would be adding the 'launchFromLara' param. Can we remove it from here?